tests/integration-tests/tests/integration/sqlite_authority_tests.rs

```diff
@@ -1343,7 +1343,7 @@ async fn test_recovery() {
 }
 
 #[tokio::test]
-async fn test_axfr() {
+async fn test_axfr_allow_all() {
```

Comment:
Would stick these renames in the previous commit introducing `AxfrPolicy`...

Reply:
Done.